pycoal/tests/mineral_test.py

```diff
@@ -48,10 +48,7 @@ def _test_classify_image_teardown():
 def test_classify_image():
```

Comment:
It would also appear that you have not yet provided a Unit test for the NN classification option. Please add a unit test.

Reply:
Sure!